frontend/src/components/pages/TeamDashboard/index.jsx

```diff
@@ -59,13 +64,19 @@ function DashboardUnconnected({
       userIds: team ? team.members.map((member) => member.userId) : [],
     }),
   };
-
+  if (activityLogDayCounts.length === 0)
+    getActivityLogCountsByDayForUsers({
+      activityLogDayCounts,
+      eventTypes,
+      userIds: team ? team.members.map((member) => member.userId) : [],
+    });
+ 
   return <Presentation {...props} />;
 }
 
 function mapStateToProps(state) {
   return {
-    teams: state.apiEntities.teams || {},
+    teams: state.apiEntities.teams,
     authUser: state.App.authUser || {},
```

Comment:
This props does not change - it does not seem to be a cause for any re-renders

Reply:
`authUser` does not seem to be a cause of any re-renders

Reply:
Neither would this:

```
    authUser: state.App.authUser || {thisObjectMakesTheCodeMessy: true},
    // unnecessary comments
   bubbles: null
```

If you remove the unnecessary code then the code will be cleaner. If you leave junk lying around just because it doesn't effect execution then it's still junk.